dotcom-rendering/src/web/lib/useBraze.ts

```diff
@@ -1,25 +1,42 @@
-import type { BrazeMessagesInterface } from '@guardian/braze-components';
-import { NullBrazeMessages } from '@guardian/braze-components/logic';
+import type {
+	BrazeCardsInterface,
+	BrazeMessagesInterface,
+} from '@guardian/braze-components/logic';
+import {
+	NullBrazeCards,
+	NullBrazeMessages,
+} from '@guardian/braze-components/logic';
 import useSWRImmutable from 'swr/immutable';
-import { buildBrazeMessages } from './braze/buildBrazeMessages';
+import { buildBrazeMessaging } from './braze/buildBrazeMessaging';
 
 /**
- * Returns brazeMessages as BrazeMessagesInterface
+ * Returns brazeMessaging as BrazeMessagesInterface and BrazeCardsInterface
+ *
+ * BrazeMessages is used to show single-impression messages (like ad impressions).
+ * In contrast, BrazeCards can prodivde persistent user notifications.
```

Comment:
I love the cSpell extension to catch these early!

```suggestion
 * In contrast, BrazeCards can provide persistent user notifications.
```